public/app/features/plugins/admin/components/InstallControls/InstallControlsButton.tsx

```diff
@@ -126,7 +131,7 @@ export function InstallControlsButton({
           onDismiss={hideConfirmModal}
         />
         <HorizontalGroup align="flex-start" width="auto" height="auto">
-          <Button variant="destructive" disabled={isUninstalling} onClick={showConfirmModal}>
+          <Button variant="destructive" disabled={isUninstalling || isUninstallingInCloud} onClick={showConfirmModal}>
```

Comment:
This expression along the previous one (line 117) makes the logic to show `isUninstalling` hard to follow because the logic is now spread in 3 different places.

Could you make a single boolean variable that holds the final calculated value and it is used here?